win32/build

```diff
@@ -1,2 +1,2 @@
 #! /bin/sh
-podman build -t docker.io/jeromerobert/wine-mingw64:1.9.0 .
+podman build -t docker.io/kbengs/wine-mingw64:1.10.0 .
```

Comment:
It may become tedious to change this back and forth, maybe we should move that all to ghcr (Github container registry)? I guess I could also build some pipelines to build and update those directly with github actions...

Reply:
Now I read your question in the top post correctly, that might make sense in the mean time.

Reply:
Yes would be good to store the images on the PDF arranger teams ghcr. I'll see if I can move this image there.